Autopilot/AttitudeManager/Src/attitudeManager.cpp

```diff
@@ -16,9 +16,9 @@ void attitudeManager::setState(attitudeState& newState)
     if (*currentState == fetchInstructionsMode::getInstance()) {
         status = AttMan::COMPLETED_CYCLE;
     } else if (*currentState == FatalFailureMode::getInstance()) {
-        status = AttMan::FAILURE_MODE;
+        status = AttMan::COMPLETED_CYCLE;
```

Comment:
I'll rectify this as well, I changed it all to COMPLETED_CYCLE for testing purposes and forgot to revert it